server/authorization.go

```diff
@@ -277,6 +279,10 @@ func addHeaderTransforms(req *envoy_auth.CheckRequest, envRequest *config.Enviro
 				}
 			}
 
+			// strip proxy base path from request path
+			addHeaderValueOption(okResponse, envoyPathHeader, envRequest.GetOperationPath(), false)
```

Comment:
This is currently done as the first so a potential `:path` override can happen subsequently. Just to make sure, do we allow this and in general do we allow customers to configure pseudo-header transformations?